client/my-sites/plans/current-plan/purchases-listing.jsx

```diff
@@ -80,6 +80,13 @@ class PurchasesListing extends Component {
 		return product.expiryMoment < moment().add( 30, 'days' );
 	}
 
+	getJetpackBackup() {
```

Comment:
I wonder if the method name could be a bit more descriptive? Is it getting the product object, or the raw product, or the purchase? Maybe `getJetpackBackupPurchase` would make more sense?